docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -181,6 +181,10 @@ These examples show two valid YAML syntax options for arrays.
 You must include the `default` namespace in the list if you want the operator to monitor both the `default` namespace and some other namespaces.
 {{% /notice %}}
 
+{{% notice note %}}
+This value is ignored if `dedicated` is set to `true`. The operator will then only manage domains in its own namespace.
```

Comment:
The operator will then only manage domains -> Then, the operator will manage only domains

Reply:
done. thanks.

Reply:
> Slight re-wording of one sentence.

Updated.

Please see a newly added section in docs-source/content/security/service-accounts.md .  Thanks.